charts/draft-store-service/values.yaml

```diff
@@ -15,6 +16,5 @@ java:
   image: 'hmctspublic.azurecr.io/draft-store/service:latest'
   keyVaults:
     "draft-store":
-      resourceGroup: draft-store-service
       secrets:
         - service-POSTGRES-PASS
```

Comment:
you do need to add the app insights key here?
AppInsightsInstrumentationKey

Reply:
will do in next PR :)